docs/content/v1/usage/event.md

```diff
@@ -3,18 +3,56 @@ title: Binding events
 weight: 5
 ---
 
-#### `@` Directive for binding Events
+#### `@` Directive for binding events
 
 ```html
 <template>
-  <button @click="sayHi" name="the button">click</button>
+  <button @click="sayHi" name="the button">Click</button>
+</template>
 
-  <script>
-    export default class extends Lego {
-      sayHi(event) {
-        alert(`You clicked to says hi! 👋🏼`)
-      }
+<script>
+  export default class extends Lego {
+    sayHi(event) {
+      alert('You clicked to says hi! 👋🏼')
     }
-  </script></template
->
+  }
+</script>
+```
+
+#### Binding custom events

```

Comment:
@vinyll This use case is not enough? It happens when an element or Lego component emit a custom event within another Lego component. Currently, it can only be listened with `addEventListener` in the `connected` hook.